docs/user-guide/ze-profiles.md

```diff
@@ -111,4 +111,20 @@ Connect your service profile with a base profile and token.
 
    The profile appears in the tree and you can now use this profile to access z/OSMF via the API Mediation Layer.
 
-For more information, see [Integrating with API Mediation Layer](https://docs.zowe.org/stable/user-guide/cli-usingcli.html#integrating-with-api-mediation-layer).
\ No newline at end of file
+For more information, see [Integrating with API Mediation Layer](https://docs.zowe.org/stable/user-guide/cli-usingcli.html#integrating-with-api-mediation-layer).
+
+### Log in to API Mediation Layer
+
+If the token for your base profile is no longer valid, you can log in again to get a new token with the Log in to API Mediation Layer feature.
+
+**Note:** The feature is only available for base profiles.
+
+**Follow these steps:**
+
+1. Open Zowe Explorer.
+2. Right-clikc your profile.
+3. Select the **Log in to API Mediation Layer** option.
```

Comment:
The new option name is **Log in to Authentication Service**.

Although, the only option for now is via the API Mediation Layer, other extenders may have a different authentication service. Perhaps this should be mentioned somewhere?

Reply:
I think that this comment should go in the release notes for the version, as well as in the doc somewhere. We'll just have to track it and update when other services are available.